torchvision/transforms/transforms.py

```diff
@@ -1040,6 +1044,9 @@ def __repr__(self):
 
 class ColorJitter(torch.nn.Module):
     """Randomly change the brightness, contrast, saturation and hue of an image.
+    The image can be a PIL Image or a Tensor, in which case it is expected
+    to have [..., H, W] shape, where ... means an arbitrary number of leading
+    dimensions. This transform does not support torchscript.
```

Comment:
```suggestion
    dimensions.
```

I think it's supported. See the tests.

Reply:
You are right! I put that there the moment I saw a jit.unused... in _check_input() I thought that means unsupported. Since private methods can support jit like this, I think some of my previous code might needs a rewrite.